### PR TITLE
Move vagrant-tramp-term key binding to correct prefix

### PR DESCRIPTION
**Pull request by rakyi**

### CODE REVIEW

File: layers/+tools/vagrant/packages.el [hunk 1]
```diff
@@ -9,7 +9,7 @@
 ;;
 ;;; License: GPLv3
 (setq vagrant-packages '(vagrant
-                           vagrant-tramp))
+                         vagrant-tramp))
```
Comment (robbyoconnor) [inline layers/+tools/vagrant/packages.el]: This change is unnecessary.

File: layers/+tools/vagrant/packages.el [hunk 2]
```diff
@@ -39,4 +39,4 @@
         (unless spacemacs--vagrant-tramp-loaded
           (vagrant-tramp-add-method)
           (setq spacemacs--vagrant-tramp-loaded t)))
-      (spacemacs/set-leader-keys "Vt" 'vagrant-tramp-term))))
+      (spacemacs/set-leader-keys "aVt" 'vagrant-tramp-term))))
```
Comment (robbyoconnor) [inline layers/+tools/vagrant/packages.el]: :+1:

File: layers/+tools/vagrant/packages.el (hunk 1 above)
Comment (rakyi) [reply layers/+tools/vagrant/packages.el]: I thought it was just an obvious indentation fix. So is the original style preferred? It doesn't look very nice to me.

**Comment by syl20bnr** [comment]
Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**